util/dvsim/dvsim.py

```diff
@@ -236,6 +302,10 @@ def parse_args():
                       help=('Prepend this string when running each tool '
                             'command.'))
 
+    disg.add_argument("--local",
+                      action='store_true',
+                      help=('Prevent copying the repo to the scratch area.'))
```

Comment:
I don't understand why this option is needed. Reading the logic above, can't a user do `env DVSIM_REMOTE_DISPATCH= util/dvsim.py --more --args --here`?

If you would prefer this to be explicit, maybe it should be the other way around: get rid of the DVSIM_REMOTE_DISPATCH environment variable and set `--remote`?

Reply:
Yes this will work, but its a little less obvious. I will remove the `--local` since we do have a [couple of options](https://github.com/lowRISC/opentitan/issues/4562#issuecomment-744817893).